Project 4/README.md

```diff
@@ -44,4 +44,48 @@ You're probably thinking, "why would I submit a level 8 kata if they're not wort
 
 It is your responsibility and the responsibility of your peers reviewing your submission in PR to determine whether your submission is ranked appropriately. In the event that consensus is reached that your kata is ranked inappropriately, you must work with your peers to revise the submission so that it is either more or less challenging, accordingly. You are not permitted to submit new problems with different strengths after PRs are open, but must instead revise your PRs. So, think hard about how challenging your submission is. 
 
-There is one other option for those desiring a different sort of challenge. If you provide alongside your SPARQL submission a translation of the same problem into SQL, complete with documentations, solution, etc. then you may receive half points extra at that kata level (rounded up). For example, if you submit a SPARQL problem that is kata rank 1 and also submit a SQL version of that same problem, you  will receive 35+18=53 points. 
+There is one other option for those desiring a different sort of challenge. If you provide alongside your SPARQL submission a translation of the same problem into SQL, complete with documentations, solution, etc. then you may receive half points extra at that kata level (rounded up). For example, if you submit a SPARQL problem that is kata rank 1 and also submit a SQL version of that same problem, you  will receive 35+18=53 points.
+
+
+
+QUESTION 1 - Kata 8
+Write a SPARQL query that provides all of the types of “things” under which Barack Obama is classified, according to DBpedia.
+
+TIPS: 
+Use {dbr:Barack_Obama} as the Subject.
+Use {rdf:type} as the Predicate.
```

Comment:
Note: Content bounded by "{...}" should be a (set of) triple(s). So here it is fine to directly say dbr:Barack_Obama or rdf:type.

Reply:
I was trying to figure out if there was some sort of convention for separating the part of the text that is supposed to be used from the rest of it (like DuCharme talks about in the book in the Preface, page XV). Do we know what the convention would be AND how to do that in GitHub Markdown?